packages/core/contracts-ovm/insured-bridge/implementation/BridgePoolFactory.sol

```diff
@@ -7,21 +7,25 @@ pragma solidity ^0.8.0;
 // of the imported DVM contracts below.
 import "./OVM_CrossDomainEnabled.sol";
 import "./BridgePoolFactoryInterface.sol";
+import "./BridgePool.sol";
 import "../../../contracts/oracle/interfaces/IdentifierWhitelistInterface.sol";
 import "../../../contracts/oracle/interfaces/FinderInterface.sol";
 import "../../../contracts/oracle/implementation/Constants.sol";
 import "../../../contracts/common/interfaces/AddressWhitelistInterface.sol";
+import "../../../contracts/common/implementation/Testable.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 /**
  * @notice Administrative contract deployed on L1 that has an implicit reference to a DepositBox. This contract is
  * responsible for deploying new BridgePools, which houses passive liquidity and enables relaying of L2 deposits.
  * @dev The owner of this contract can call permissioned functions on the L2 DepositBox.
  */
-contract BridgePoolFactory is BridgePoolFactoryInterface, Ownable, OVM_CrossDomainEnabled {
+contract BridgePoolFactory is BridgePoolFactoryInterface, Ownable, Testable, OVM_CrossDomainEnabled {
     // Finder used to point to latest OptimisticOracle and other DVM contracts.
     address public override finder;
 
+    address public timer;
```

Comment:
you dont need to store the timer address, do you? why would you need this and cant access it from the inherited `Testable`

Reply:
True